articles/managed-grafana/tutorial-mpe-oss-prometheus.md

```diff
@@ -143,7 +143,7 @@ spec:
     - name: http-web
       protocol: TCP
       port: 9090
-      targetPort: 9090
+      targetPort: http-web # this information should be the same as the selector port configuration
```

Comment:
I do not understand why your existing prometheus's operator service tartgetPort is this. But 9090 is the default value from upstream helm chart. 

[http-web](https://github.com/prometheus-community/helm-charts/blob/67e796b1ee66ac9d5c9fa4aa3305a3c7f17daa0a/charts/kube-prometheus-stack/values.yaml#L3051)

Reply:
Hello, @weng5e .

When i created this PR, the targetPort was "http-web".
Today i recreated the service after "helm repo update" and the targetPorts are "9090".

Maybe i was using an old repo.

![image](https://github.com/MicrosoftDocs/azure-docs/assets/38931377/da269b64-8f71-4945-856e-f627873bec9b)


New deployment today.
![image](https://github.com/MicrosoftDocs/azure-docs/assets/38931377/26aaf1fa-f168-48d2-91da-efb203c7e93f)


Please, disregard this PR, then :)

Thank you